cmd/cmd.go

```diff
@@ -44,7 +44,7 @@ const (
 	// FlagSlowLogFile is the name of slow-log-file flag.
 	FlagSlowLogFile = "slow-log-file"
 	// FlagRedactLog is whether to redact sensitive information in log.
-	FlagRedactLog = "redact-log"
+	FlagRedactLog = "redact-info-log"
```

Comment:
🤔️ can we keep both "redact-log" and "redact-info-log" for compatibility.

Reply:
Seems it will be release in 5.0.0-rc.

Reply:
I have already added the compatibility support. PTAL again.

Reply:
i don't think we need to support `--redact-log` if it does not appear in any GA version.